compiler/rustc_lint/src/context.rs

```diff
@@ -952,6 +952,10 @@ pub trait LintContext: Sized {
                     db.span_label(first_reexport_span, format!("the name `{}` in the {} namespace is first re-exported here", name, namespace));
                     db.span_label(duplicate_reexport_span, format!("but the name `{}` in the {} namespace is also re-exported here", name, namespace));
                 }
+                BuiltinLintDiagnostics::HiddenGlobReexports { name, namespace, glob_reexport_span, private_item_span } => {
+                    db.span_label(glob_reexport_span, format!("the name `{}` in the {} namespace is introduced by the glob reexport here", name, namespace));
+                    db.span_label(private_item_span, format!("but the private item here shadows the name `{}` in the {} namespace", name, namespace));
```

Comment:
```suggestion
                    db.span_label(private_item_span, format!("but the private item here shadows it"));
```
(To avoid tautology.)